### DISCUSSION TITLE
Patches install dates not showing

### DISCUSSION THREAD

**Discussion by ninjamonkey198206** [amidaware/tacticalrmm#998] [Q&A]
Everything else seems to be working, so I figured I'd ask and see if anyone else is seeing this before I post a bug report.

I never really paid attention to it until recently, so I don't know if it's a recent development or not.

**Comment by wh1te909** [comment]
the install date will only show if the patch was installed by tacticalrmm

**Reply by ninjamonkey198206** [reply]
Which it should consider to be the case if automatic updates are disabled, the updates in question are manually approved, and told to install, correct?

I honestly don't know, just trying to sus out if this is my lack of knowledge, a possible issue, or working as intended.
